programs/stake/src/stake_instruction.rs

```diff
@@ -288,7 +288,6 @@ pub fn split_with_seed(
             std::mem::size_of::<StakeState>() as u64,
             &id(),
         ),
-        system_instruction::assign_with_seed(split_stake_pubkey, base, seed, &id()),
```

Comment:
Landmines everywhere!

https://github.com/solana-labs/solana/blob/7bb5f24274d1412e6f78616962d020a3de7aa841/runtime/src/system_instruction_processor.rs#L391-L404

Reply:
Seriously. You should see my most recent commit :sweat_smile: